tests/testthat/test-check.R

```diff
@@ -47,3 +51,24 @@ test_that("check_urls reports problems", {
   pkg <- test_path("assets/cname")
   expect_snapshot(check_urls(pkg), error = TRUE)
 })
+
+# check favicons --------------------------------------------------------------
+
+test_that("check_favicons reports problems", {
+  pkg <- local_pkgdown_site()
+  # no logo no problems
```

Comment:
```suggestion

  # no logo no problems
```

the rest of this test is so beautifully organized :star_struck: